integration_test/workflow_tests/test_packages/ingest/test_no_granules.py

```diff
@@ -44,7 +44,7 @@ def test_no_granules_passes(self):
             }
 
             expected_output = {
-                "payload": {"granules": [], "copied_to_glacier": []},
+                "payload": {"granules": [], "copied_to_archive": []},
```

Comment:
Thinking this should be `copied_to_orca` as this is forward facing to the user and will better define what was sent to ORCA.

Reply:
If they're receiving output from an Orca lambda, they should already know the context. My understanding was the we were only worried about what they say when looking through AWS resources.